src/System.Drawing.Common/src/System/Drawing/Gdiplus.cs

```diff
@@ -856,6 +856,14 @@ public static IntPtr GlobalAlloc(int uFlags, uint dwBytes)
         [DllImport(ExternDll.Kernel32)]
         static internal extern void ZeroMemory(IntPtr destination, UIntPtr length);
```

Comment:
Can you remind us of the issue? If ZeroMemory out of kernel32 is an issue, should this extern declaration be removed?

Reply:
We were getting this exception `System.EntryPointNotFoundException : Unable to find an entry point named 'ZeroMemory' in DLL 'kernel32.dll'`  call this method. 

I was not sure if I can remove it or not.